PWGHF/Utils/utilsEvSelHf.h

```diff
@@ -142,9 +158,9 @@ struct HfEventSelection : o2::framework::ConfigurableGroup {
   /// \param registry reference to the histogram registry needed for zorro
   /// \return bitmask with the event selection criteria not satisfied by the collision
   template <bool useEvSel, o2::hf_centrality::CentralityEstimator centEstimator, typename BCs, typename Coll>
-  uint16_t getHfCollisionRejectionMask(const Coll& collision, float& centrality, o2::framework::Service<o2::ccdb::BasicCCDBManager> const& ccdb, o2::framework::HistogramRegistry& registry)
+  uint32_t getHfCollisionRejectionMask(const Coll& collision, float& centrality, o2::framework::Service<o2::ccdb::BasicCCDBManager> const& ccdb, o2::framework::HistogramRegistry& registry)
   {
-    uint16_t rejectionMask{0}; // 16 bits, in case new ev. selections will be added
+    uint32_t rejectionMask{0}; // 16 bits, in case new ev. selections will be added
```

Comment:
If you change the type of the mask, you should modify it everywhere it's used. For instance, this is also used in the https://github.com/AliceO2Group/O2Physics/blob/master/PWGHF/TableProducer/trackIndexSkimCreator.cxx task and stored in this table https://github.com/AliceO2Group/O2Physics/blob/a2a96905974a13d9c77cb15b0f3f2eeac487e3f2/PWGHF/DataModel/CandidateReconstructionTables.h#L194 (which also has to be modified, otherwise it gets truncated). 
You should carefully check everywhere it is used and make sure that it is updated to 32 bits everywhere

Reply:
Why is making the type bigger a problem? When reading data with `uint16_t`, are they not implicitly converted to `uint32_t`?